LICENSE

```diff
@@ -188,3 +190,52 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+# pgsodium License
```

Comment:
is this necessary if the dep is getting pulled out?

Reply:
Yeah we use pgsodium code in pgsodium.c and pgsodium.h; we can do away with them once we move from libsodium (still needed to decrypt existing secrets)